README.md

```diff
@@ -1,33 +1,108 @@
 # Cloud in a Box
 
-**The secrets are stored in plain text and are not secure. Do not use for public
-accessible systems.**
+Cloud in a Box is a minimalistic installation of a OSISM with only services which are needed to make it run.
+For more information you can consult the [Testbed](https://docs.osism.tech/testbed/index.html) documentation.
+If you want to have a full installation please have a look at the [OSISM](https://docs.osism.tech/) documentation.
 
-## Assumptions
+> **Warning:**
+> The secrets are stored in plain text and are not secure. Do not use for public
+> accessible systems.
 
-* the 1st blockdevice is available as ``/dev/sda`` or ``/dev/nvme0n1``
+## Requirements
 
-## Download
+* The first blockdevice is available as `/dev/sda` or `/dev/nvme0n1`.
+* You need to have at least a second blockdevice which is from the same type as the first, empty and usable.
+* USB stick with at least 2GB capacity.
+* CPU: 16 core, 32 threads
```

Comment:
16 cores (note the "s")
(Side question: Do we really need that many cores?  I would suspect a system with 8 cores / 16 threads to work just fine, just getting slower once you start too many VMs. Wording could be 8 cores / 16 threads -- twice that is recommended for good performance.)

Reply:
We were advised to use the stats of the reference hardware.

Reply:
We tested everything with the cloud-in-a-box hardware (1 socket with 16 physical cores). Maybe we still have a good UX with less than 16 cores, maybe not. I don't know. I opened https://github.com/osism/cloud-in-a-box/issues/27 to test the deployment & UX with less cores in the future. We can then adjust the minimal hardware recommendations in the future.